code/game/machinery/computer/communications.dm

```diff
@@ -110,7 +110,7 @@ var/shuttle_call/shuttle_calls[0]
 				var/obj/item/device/pda/pda = I
 				I = pda.id
 			if (I && istype(I))
-				if(access_captain in I.access || access_heads in I.access) //Let heads change the alert level.
```

Comment:
Is that necessary? the access_captain call is redundant seeing as how the Captain has access_heads.